src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -23,7 +23,8 @@ namespace NuGet.Commands.Restore.Utility
 {
     internal class AuditUtility
     {
-        private readonly ProjectModel.RestoreAuditProperties? _restoreAuditProperties;
+        private readonly IEnumerable<DownloadDependency> _downloadDependencies;
```

Comment:
It's a good practice to add new types at the bottom of the field declarations unless the new type is clearly the most important one in this class.